### ISSUE TITLE
Api 4

### ISSUE THREAD

**Issue by ultimate31012** [jojoe77777/Slapper#326]
Pls update to api 4

**Comment by Triority2011** [comment]
yes pls do that, the plugin is essential to alot of peoples servers

**Comment by YT-Duckyyy** [comment]
Yes, please update it ASAP. Many servers are waiting and we rely on this plugin

**Comment by encyptAES** [comment]
https://github.com/vecnavium-pm-pl/Slapper
